### PR TITLE
Parse RETURNS schema in CREATE MODEL

### PR DESCRIPTION
**Pull request by eddyxu**
Support `RETURNS dataType` in CREATE MODEL

### CODE REVIEW

**Comment by eddyxu** [comment]
This closes #211

File: python/tests/spark/test_functions.py [hunk 1]
```diff
@@ -17,13 +17,12 @@
 import os
 from io import BytesIO
 from pathlib import Path
-import pytest
```
Comment (eddyxu) [inline python/tests/spark/test_functions.py:20]: side effect after running `make fix`

**Review by da-tubi — APPROVED** [review]
LGTM

**Comment by eddyxu** [comment]
I will merge this with Da's approval.

**Pull request merged**